sprokit/src/applets/pipeline_runner.cxx

```diff
@@ -65,32 +69,38 @@ add_command_options()
 
 // ----------------------------------------------------------------------------
 int
-pipeline_runner::
-run()
+pipeline_runner
+::run()
 {
   const std::string opt_app_name = applet_name();
 
   auto& cmd_args = command_args();
 
-  if ( cmd_args["help"].as<bool>() )
+  if( cmd_args[ "help" ].as< bool >() )
   {
     std::cout << m_cmd_options->help();
     return EXIT_SUCCESS;
   }
 
   // Load all known modules
-  kwiver::vital::plugin_manager& vpm = kwiver::vital::plugin_manager::instance();
+  kwiver::vital::plugin_manager& vpm =
+    kwiver::vital::plugin_manager::instance();
   vpm.load_all_plugins();
 
   sprokit::pipeline_builder builder;
 
-  // Add search path to builder.
-  if ( cmd_args.count("include") > 0 )
+  // Add standard search locations
+  const std::string prefix = kwiver::vital::get_executable_path() + "/..";
+  builder.add_search_path( kwiver::vital::kwiver_config_file_paths( prefix ) );
+
+  // Add user-provided paths
+  if( cmd_args.count( "include" ) > 0 )
```

Comment:
I'm thinking that the user provided paths probably should go before the standard paths.  This way the user paths have priority over the standard paths and you can override a system file with a local one.  I believe that the order we add search paths is the order they are searched, but I'm not 100% sure of this.

Reply:
This sounds correct, at least for config files. Updated.